front-end/src/pages/Admin/AdminMaterials/MaterialForm/MaterialForm.jsx

```diff
@@ -123,8 +130,16 @@ const CreateMaterial = ({ material, levels, create }) => {
         setIsLoading(true);
         await createMaterial(materialData);
         setIsLoading(false);
+        snackbarShowMessage({
+          message: 'Material saved',
+          severity: 'success',
+        });
       } else {
```

Comment:
Maybe you should add loader when admin update material, just like when admin create it. Also you could try move loader logic to avoid duplication in if/else statement.

Reply:
done